dnsmasq.conf

```diff
@@ -0,0 +1,24 @@
+interface=provisioning
```

Comment:
Hi @bfournie interface _provisioning_ doesn't exist on master nodes, can it be made parameterized?
/cc @hardys @mcornea

Reply:
I think it should, yes

Reply:
Good point, added INTERFACE env variable to allow configuration.